systemservices/marketplace/src/contracts/manifest.ts

```diff
@@ -1,5 +1,10 @@
+import { Validator } from "jsonschema"
 import Request from "request-promise-native"
-import { Manifest } from "../types/manifest";
+import { Manifest } from "../types/manifest"
+import manifestSchema from '../types/schema/manifest.json'
+import definitionSchema from '../types/schema/definition.json'
```

Comment:
```suggestion
import serviceSchema from '../types/schema/service.json'
```

Could you rename definition for service? It's easier to guess what it is.

Reply:
done